base/loading.jl

```diff
@@ -1303,16 +1303,20 @@ end
 
 const MAX_NUM_PRECOMPILE_FILES = Ref(10)
 
-function compilecache(pkg::PkgId, path::String, internal_stderr::IO = stderr, internal_stdout::IO = stdout)
+function compilecache(pkg::PkgId, path::String, internal_stderr::IO = stderr, internal_stdout::IO = stdout,
+        ignore_loaded_modules::Bool = true)
```

Comment:
Can you add enough spaces to match this up with the rest of the args?

Reply:
yep! done